theano/tensor/elemwise.py

```diff
@@ -2029,6 +2017,7 @@ class Prod(CAReduceDtype):
     input.
 
     """
+    __props__ = ("axis", "dtype", "acc_dtype")
```

Comment:
Add a comment that we don't want to add "no_zeros_in_input" as it don't change what this op compute. It only change what the grad change.